### DISCUSSION TITLE
Sign up for the GitHub Customer Research Panel!

### DISCUSSION THREAD

**Discussion by gracevor** [community/community#74753] [Announcements]
## Sign up for GitHub’s Customer Research Panel! 

[Click here to sign up!](https://github.co/GH_Research_Panel)

### Why sign up for GitHub’s Customer Research Panel? 

Hearing directly from our customers helps GitHub make sure we understand your wants and needs and build products that match the way you think and work. You may also: 

* Get early access to things design and product teams are working on (including alphas, betas, and the like)
* Have direct access to designers, product managers, and other Hubbers working on customer solutions / new features 
* Influence GitHub product strategy and decisions through your feedback 

[Click here](https://cxr.github.com/) to learn more about participating in GitHub Customer Research. Happy Universe! 

![860 x 490_Instagram_Hero_banner](https://github.com/community/community/assets/2140740/6c97a81a-e443-4d92-80c9-f74aae889b8a)

**Comment by Maikki86** [comment]
Hei Miten haen seuraavaan koulutukseen :Gi1tHub Actions -koulutus?

**Reply by GaoYingZhang** [reply]
Hi l am GaoYing

**Comment by guardnow** [comment]
- [x]

**Comment by guardnow** [comment]
@@

**Reply by guardnow** [reply]
Please don't mess around with my stuff

**Comment by Miguel-Sanz** [comment]
Just wanted to share some exciting updates from GitHub! 🎉 There are new and interesting things happening that caught my attention, and I couldn't help but celebrate with all of you.

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
